### ISSUE TITLE
Appium should always send back original sessionId not chromedriver session Id

### ISSUE THREAD

**Issue by moizjv** [appium/appium#5305]
```
2015-06-24 23:18:20:853 - info: <-- POST /wd/hub/session/404f5f40-bd3b-4491-b345-d336e042517a/context 200 978.626 ms - 76 {"status":0,"value":null,"sessionId":"404f5f40-bd3b-4491-b345-d336e042517a"}
2015-06-24 23:18:20:888 - info: --> GET /wd/hub/session/404f5f40-bd3b-4491-b345-d336e042517a/context {}
2015-06-24 23:18:20:889 - info: [debug] Responding to client with success: {"status":0,"value":"WEBVIEW_io.selendroid.testapp","sessionId":"404f5f40-bd3b-4491-b345-d336e042517a"}
2015-06-24 23:18:20:890 - info: <-- GET /wd/hub/session/404f5f40-bd3b-4491-b345-d336e042517a/context 200 1.161 ms - 103 {"status":0,"value":"WEBVIEW_io.selendroid.testapp","sessionId":"404f5f40-bd3b-4491-b345-d336e042517a"}
2015-06-24 23:19:10:952 - info: --> POST /wd/hub/session/404f5f40-bd3b-4491-b345-d336e042517a/refresh {}
2015-06-24 23:19:10:955 - info: Proxying [POST /wd/hub/session/404f5f40-bd3b-4491-b345-d336e042517a/refresh] to [POST http://127.0.0.1:9515/wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/refresh] with body: {}
2015-06-24 23:19:11:108 - info: Got response with status 200: {"sessionId":"eedd7a9d105a66de2d4693fa6d3db165","status":0,"value":null}
2015-06-24 23:19:11:110 - info: <-- POST /wd/hub/session/404f5f40-bd3b-4491-b345-d336e042517a/refresh 200 157.263 ms - 72 
2015-06-24 23:19:11:192 - info: --> POST /wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/element {"using":"id","value":"name_input"}
2015-06-24 23:19:11:193 - info: Proxying [POST /wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/element] to [POST http://127.0.0.1:9515/wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/element] with body: {"using":"id","value":"name_input"}
2015-06-24 23:19:11:291 - info: Got response with status 200: {"sessionId":"eedd7a9d105a66de2d4693fa6d3db165","status":0,"value":{"ELEMENT":"0.6228732399176806-1"}}
2015-06-24 23:19:11:292 - info: <-- POST /wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/element 200 99.686 ms - 102 
2015-06-24 23:19:11:358 - info: --> POST /wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/element/0.6228732399176806-1/value {"id":"0.6228732399176806-1","value":["A","l","b","e","d","i","t","h"]}
2015-06-24 23:19:11:358 - info: Proxying [POST /wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/element/0.6228732399176806-1/value] to [POST http://127.0.0.1:9515/wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/element/0.6228732399176806-1/value] with body: {"id":"0.6228732399176806-1","value":["A","l","b","e","d","i","t","h"]}
2015-06-24 23:19:13:412 - info: Got response with status 200: {"sessionId":"eedd7a9d105a66de2d4693fa6d3db165","status":0,"value":null}
2015-06-24 23:19:13:413 - info: <-- POST /wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/element/0.6228732399176806-1/value 200 2055.470 ms - 72 
2015-06-24 23:19:23:790 - info: --> POST /wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/context {"name":"NATIVE_APP"}
2015-06-24 23:19:23:792 - info: <-- POST /wd/hub/session/eedd7a9d105a66de2d4693fa6d3db165/context 404 2.363 ms - 40 
```

**Comment by hannesstruss** [comment]
Hi, I originally reported this to the Saucelabs support. If I can be of any help, just ping me here!

**Comment by moizjv** [comment]
@hannesstruss thanks for raising this issue. I have raised a pull request for this bug.
https://github.com/appium/jsonwp-proxy/pull/2 will update when we fix it.

**Comment by moizjv** [comment]
@hannesstruss this issue is now fixed in 1.4.7 and is also available on sauce. Can you retry now? Thanks
